### ISSUE TITLE
What happened to pfsense true bridge-mode?

### ISSUE THREAD

**Issue by Martyfree123** [aus/proxyplease#2]
Hey, this is weird, but what happened to your pfsense - at&t bridge mode repo? I just followed the guide the other day and came back to look at something and now it's gone? (Sorry for the report issue, only way I could find to contact you.)

**Comment by briangordon** [comment]
Yeah, luckily I had downloaded it before it went away, but I'm in the middle of trying to set it up and all the discussion in the github issues that I was referring to has disappeared :(

**Comment by Martyfree123** [comment]
Any chance I can get a copy?

**Comment by owenthewizard** [comment]
I speculate that AT&T sent a cease and desist.

**Comment by Martyfree123** [comment]
Won't let that stop me :)
